src/services/ProjectTokenHistoricalPriceService.ts

```diff
@@ -243,7 +243,7 @@ export class ProjectTokenHistoricalPriceService {
     const prices = await this.getFundingTokenUSDPricesByID(
       lbpMgr.fundingTokenInfo.id,
       endTimeSeconds,
-      startTimeSeconds,
+      startTimeSeconds < currentTime ? startTimeSeconds : Math.floor(currentTime) - 3600,
```

Comment:
Maybe is fine as is, but my paranoia always makes me use parens around the expressions in terniary expressions:

`(startTimeSeconds < currentTime) ? startTimeSeconds : (Math.floor(currentTime) - 3600)`